pkgs/applications/networking/instant-messengers/signalbackup-tools/default.nix

```diff
@@ -0,0 +1,37 @@
+{ lib, stdenv, fetchFromGitHub, openssl, sqlite }:
+
+stdenv.mkDerivation rec {
+  pname = "signalbackup-tools";
+  version = "20220107";
+
+  src = fetchFromGitHub {
+    owner = "bepaald";
+    repo = pname;
+    rev = version;
+    sha256 = "sB8/xQgSORtwupcwSejKUhHoz04exdYS0ymefw9wXDQ=";
+  };
+
+  # Remove when Apple SDK is >= 10.13
+  patches = lib.optional (stdenv.system == "x86_64-darwin") ./apple-sdk-missing-utimensat.patch;
```

Comment:
So the package does build on `aarch64-darwin` with the patch applied. However I think it's probably best to only apply the patch when it's needed so that, if the patch has some unintended consequence, it doesn't affect platforms that don't require the patch in the first place.

Reply:
but then it will most likely be forgotten when the package is updated from someone not being on x86_64-darwin.

Reply:
I'll do my best to keep and eye on it, and remove the patch once #101229 lands.